### DISCUSSION TITLE
Is the jxl compression format used in the software lossless compression ？

### DISCUSSION THREAD

**Discussion by Linxuanhua1** [d2phap/ImageGlass#1996] [Q&A]
I use the jxl compression in XnViewer MP to compress the image and the size just reduce about 20%.
I use saving as jxl in Imageglass and the size reduce about 80%, amazing!

**Comment by d2phap** [comment]
Hi @Linxuanhua1, you can use "Magick.NET Lossless Compression" tool (in Main Menu > Tools) to compress it. "Save as" does not.

**Reply by Linxuanhua1** [reply]
OK, OK, I know, thank you!
